fhir-smart/src/main/java/org/linuxforhealth/fhir/smart/AuthzPolicyEnforcementPersistenceInterceptor.java

```diff
@@ -75,7 +75,7 @@
  * <p><a href="http://hl7.org/fhir/smart-app-launch/1.0.0/scopes-and-launch-context/index.html">SMART App Launch: Scopes and Launch Context</a>
  * defines the following pattern for the OAuth 2.0 scopes expected in the JWT:
  * <pre>
- * ( 'patient' | 'user' ) '/' ( fhir-resource | '*' ) '.' ( 'read' | 'write' | '*' )`
+ * ( 'patient' | 'user' | 'system' ) '/' ( fhir-resource | '*' ) '.' ( 'read' | 'write' | '*' )`
```

Comment:
```suggestion
 * ( 'patient' | 'user' ) '/' ( fhir-resource | '*' ) '.' ( 'read' | 'write' | '*' )`
```

Reply:
i shouldn't have included this change.  It clearly says this is whats defined by SMART App Launch.  And it already says SMART Backend Servcies extends that to include context type 'system' just belown.